src/envo/envo-edit.owl

```diff
@@ -44772,6 +44776,51 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasBroadSynony
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03600090> "Definition from 2021-08-05 call with Buttigieg, Duncan and Miller")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03600090> "alcohol fermentation pit")
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03600090> <http://purl.obolibrary.org/obo/ENVO_03600039>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03600091> (clay fermentation pit)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03600091> "A fermentation pit which is constructed from clay.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600091> "https://orcid.org/0000-0001-9076-6066")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600091> "https://orcid.org/0000-0001-9625-1899")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600091> "https://orcid.org/0000-0002-4366-3088")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03600091> "2021-08-09T15:00:00.000Z")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03600091> "Definition from 2021-08-05 call with Buttigieg, Duncan and Miller")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03600091> "clay fermentation pit")
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03600091> <http://purl.obolibrary.org/obo/ENVO_01001871>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03600092> (artificial pond)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03600092> "A pond which has been constructed by humans or their technologies.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600092> "https://orcid.org/0000-0001-9076-6066")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600092> "https://orcid.org/0000-0001-9625-1899")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03600092> "https://orcid.org/0000-0002-4366-3088")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03600092> "2021-08-09T15:00:00.000Z")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03600092> "Definition from 2021-08-05 call with Buttigieg, Duncan and Miller")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03600092> "artificial pond")
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03600092> <http://purl.obolibrary.org/obo/ENVO_00000033>)
```

Comment:
I don't see the axiom about being the output of a human-led construction process - that would allow it to be reasoned under the respective branch.